locale/en/corteza-webapp-compose/general.yaml

```diff
@@ -55,6 +55,7 @@ label:
   processing: Processing...
   record: Record
   recordPage: Record page
+  recordPageLink: Go to Record page
```

Comment:
Probably best to name it view record, since you're not just going to record page you're actually viewing record here